Extensions/LocationTools.json

```diff
@@ -0,0 +1,328 @@
+{
+  "author": "Arthur Pacaud (arthuro555)",
+  "description": "This extension allows to \n- Get the location of the game page\n- Reload the page\n- Redirect to another page\n- Get a specific part of an URL\n- Get an HTTP GET parameter of a URL\n- Modify a specific property of a URL",
+  "extensionNamespace": "",
+  "fullName": "Location tools (Web browser)",
```

Comment:
Location is too much of a term that comes from "window.location".
If I talk about Location in the usual world, people will understand it as "Geolocation", your location in space, the street where you are, the restaurant you're in...

I think this should rather be something like "URL tools" or "Webpage URL tools" to be very precise.